docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -14,15 +14,10 @@ Managing workflows can be complicated, as you may be chaining together many step
 
 What is a tool?
 ---------------
```

Comment:
Although I did ask for changes in the other breakdowns of tool definitions, I think this particular one is excellent for this introduction page.